contracts/src/challenge/SymChallenge.sol

```diff
@@ -55,26 +58,30 @@ contract SymChallenge is ChallengeBase, ISymChallenge {
     }
 
     /**
-     * @notice Initializes contract.
+     * @notice Initializes contract. Called by `Rollup.sol` when a challenge is initiated. Challenge requires additional inputs from defender and challenger to be executed.
      * @param _defender Defending party.
      * @param _challenger Challenging party. Challenger starts.
      * @param _verifier Address of the verifier contract.
      * @param _daProvider DA provider.
      * @param _resultReceiver Address of contract that will receive the outcome (via callback `completeChallenge`).
      * @param _startStateHash Commitment to agreed-upon start state.
-     * @param _endStateHash Commitment to disagreed-upon end state being challenged.
+     * @param _endStateDefenseHash Proposed DA end state hash of the defender.
+     * @param _endStateChallengeHash Proposed DA end state hash of the challenger.
+     * @param challengePeriod The duration that challenger/defender has for continuing the challenge
      */
+
```

Comment:
nit: extraneous newline between documentation and fn definition

Reply:
Fixed